dev/build.pii.package/build.gradle

```diff
@@ -15,25 +15,25 @@ task zipMessageFiles(type: Zip) {
   dependsOn parent.subprojects.copyPiiFiles
   archiveBaseName = 'openPII'
   from "${projectDir}/nlssrc"
-  destinationDirectory = distsDirectory
+  destinationDirectory = layout.buildDirectory.dir("distributions")
 }
 
 task zipMessageXmlFiles(type: Zip) {
   archiveBaseName = 'openPIIXml'
   from "${projectDir}/nlsxml"
-  destinationDirectory = distsDirectory
+  destinationDirectory = layout.buildDirectory.dir("distributions")
 }
 
 publishing {
   publications {
     maven(MavenPublication) {
-      artifactId 'openPII'
-      version project.version
```

Comment:
Does `project.` matter on the version too? There's a handful of these.

Reply:
This isn't in a doLast so it isn't an issue.  We need to use project.version to specify what version it is.